dogs.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+  <meta charset="utf-8">
+  <title>Adopt-a-dog | Dog Adoption</title>
+  <link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+  <div class="header-img">
+    <a href="index.html">
+    <img class="logo" src="images/adoptadog-logo.svg" alt="Adopt-a-dog logo.">
+    </a>
+    <h2 class="tagline">Seattle's premiere dog adoption agency</h2>
+    <div class="cart-total">
+      <img class="cart-icon" src="images/cart-icon.svg" alt="Shopping cart icon.">
+      $1234.56
+      <p><a class="checkout-btn" href="checkout.html">Checkout</a></p>
+    </div>
+  </div>
+
+  <div class="navbar">
+    <a class="navlink" href="index.html">Home</a>/
+    <a class="navlink" href="dogs.html">Dogs</a>/
+    <a class="navlink" href="blog.html">Blog</a>
+  </div>
+
+<div class="centering-container">
+
+    <div class="dogs-list-container">
+
+      <h1>Here are all of our Dogs!</h1>
+
+      <div class="doggo-card">
+        <img class="card-img" src="images/murphy-card.jpg" alt="">
+        <div class="card-text">
+        <h4>Murphy</h4>
```

Comment:
Anything nested in the dif needs to be indented in

Reply:
Done.